python/cugraph/traversal/__init__.py

```diff
@@ -11,8 +11,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import cudf
 from cugraph.traversal.bfs import bfs
 from cugraph.traversal.bfs import bfs_edges
-from cugraph.traversal.sssp import sssp
-from cugraph.traversal.sssp import shortest_path
-from cugraph.traversal.sssp import filter_unreachable
+from cugraph.traversal.sssp import (
+    sssp,
+    shortest_path,
+    filter_unreachable
+)
+
+
+# TODO: need to add docs, test with cupy matrix and networkx graphs
+def shortest_path_length(G, source, target=None):
```

Comment:
In addition to docs, please consider transferring this function to `sssp.py` for consistency